### PR TITLE
Add new commands and features | Version 3.2

### LINKED ISSUE #152: Add new commands, add new features to now playing command
New commands to add:
1. Shuffle
2. Repeat Track
3. #153 
4. Make bot self deaf on join

When Now Playing is sent or message saying new song is playing, add buttons to it that allow to skip, stop or pause.

### PR DESCRIPTION
**Pull request by AjayACST**
Closes #152

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -172,9 +221,15 @@ client.manager = new Manager({
     const response = await client.manager.search(serverQueue.songs[0].url);
     player.play(response.tracks[0]);
   })
-  .on("socketClosed", async (player) => {
```
Comment (Darkflame72) [inline index.js:175]: Is this event still used anywhere as it has been removed?

**Review by Darkflame72 — CHANGES_REQUESTED** [review]
Looks all good just one comment before approval

File: index.js (hunk 1 above)
Comment (AjayACST) [reply index.js:175]: @Darkflame72 That bit of code was removed from the file. I'm not sure why it's showing here but if you go to the branch it isn't there

Comment (AjayACST) [reply index.js:175]: No it isn't. It has been replaced by playerMoved, which is emitted on disconnect and move to another channel

**Review by Darkflame72 — APPROVED** [review]

**Pull request merged**